openedx/core/djangoapps/theming/tests/test_helpers.py

```diff
@@ -86,130 +86,37 @@ def test_is_comprehensive_theming_enabled(self):
             "openedx.core.djangoapps.theming.helpers.current_request_has_associated_site_theme",
             Mock(return_value=True),
         ):
-            with patch(
-                "openedx.core.djangoapps.theming.helpers.microsite.is_request_in_microsite",
-                Mock(return_value=True),
-            ):
-                self.assertTrue(theming_helpers.is_comprehensive_theming_enabled())
-
-        # Theming is enabled, there is no SiteTheme record but there is microsite configuration for the current site.
-        with patch(
-            "openedx.core.djangoapps.theming.helpers.current_request_has_associated_site_theme",
-            Mock(return_value=False),
-        ):
-            with patch(
-                "openedx.core.djangoapps.theming.helpers.microsite.is_request_in_microsite",
-                Mock(return_value=True),
-            ):
-                self.assertFalse(theming_helpers.is_comprehensive_theming_enabled())
+            self.assertTrue(theming_helpers.is_comprehensive_theming_enabled())
 
         # Theming is enabled, there is neither a SiteTheme record nor microsite configuration for the current site.
```

Comment:
Do you want to nix the microsite reference from this comment?  Also mentioned in a few more comments below.